.travis.yml

```diff
@@ -1,6 +1,24 @@
 language: c
-install: wget https://raw.githubusercontent.com/ocaml/ocaml-travisci-skeleton/master/.travis-opam.sh
-script: bash -ex .travis-opam.sh
-sudo: true
+sudo: false
+services:
+    - docker
+install:
+    - wget https://raw.githubusercontent.com/ocaml/ocaml-ci-scripts/master/.travis-docker.sh
+script:
+    - bash -ex .travis-docker.sh
 env:
-        - OCAML_VERSION=4.02 PACKAGE=xapi-rrdd-plugin EXTRA_REMOTES=git://github.com/xapi-project/opam-repo-dev
+    global:
+        - OCAML_VERSION=4.04.2
+        - DISTRO=debian-stable
+        - PINS="rrdd-plugin:. xapi-rrdd-plugin:."
+        - PACKAGE=rrdd-plugin
+    matrix:
+        # We need to pass some Travis environment variables to the container to
+        # enable uploading to coveralls and detection of Travis CI.
+        # Also, set TESTS to false to avoid running them twice.
```

Comment:
So, just to clarify, these are TODOs for the future?

Reply:
No, this is just copy paste of another travis test, removing coverage and not thinking to much about it as long as it does the right thing :P